src/pages/Detail/Detail.js

```diff
@@ -32,25 +42,45 @@ const Detail = () => {
     locationLongitude,
     amenities,
     studioImages,
-    booking_number,
+    ratingCount,
+    averageRating,
   } = detailsData;
 
   // 추후 mockData 통신
+  // useEffect(() => {
+  //   fetch('/data/detailData.json')
+  //     .then(res => res.json())
+  //     .then(data => {
+  //       setDetailsData(data.data);
+  //     });
+  // }, []);
+
   useEffect(() => {
-    fetch(`${process.env.REACT_APP_SERVER_HOST}/studios/details`)
+    fetch(`${process.env.REACT_APP_SERVER_HOST}/studios/details/${id}`)
       .then(res => res.json())
       .then(data => {
         setDetailsData(data.data);
       });
   }, []);
 
-  // useEffect(() => {
-  //   fetch(`http://10.58.52.71:8000/studios/details/${id}`)
-  //     .then(res => res.json())
-  //     .then(data => {
-  //       setDetailsData(data.data);
-  //     });
-  // }, []);
+  useEffect(() => {
+    fetch(`${process.env.REACT_APP_SERVER_HOST}/bookings/time/${id}`)
+      .then(res => res.json())
+      .then(data => {
+        setBookingTime(data.data);
+      });
+  }, []);
+
+  const sortTime = () => {
+    bookingTime.map(time => {
+      bookedArray.push(time.timeSlotId);
+    });
+  };
+  sortTime();
+
+  const addToRecoil = () => {
+    setBookingArray([...bookedArray]);
+  };
```

Comment:
```suggestion
  useEffect(() => {
    fetch(`${process.env.REACT_APP_SERVER_HOST}/bookings/time/${id}`)
      .then(res => res.json())
      .then(data => {
        setBookingTime(data.data);
        bookedArray = data.data.map(time => {return time.timeSlotId});
      });
  }, []);
```
마운트 시점부터 해당 값을 넣고 싶다면 받아올 때 넣어주는게 좋을 것 같아요!
아래를 보니 addToRecoil은 사용이 되는 함수라서
addToRecoil은 살려야겠네요!